### PR TITLE
Add resolver.co.uk

### PR DESCRIPTION
**Pull request by ohheyrj**
Added resolver.co.uk, complain handling assistant. Alexa rank:

image

### CODE REVIEW

File: _data/other.yml [hunk 1]
```diff
@@ -470,6 +470,12 @@ websites:
     doc: https://feedback.repairshopr.com/knowledgebase/articles/928266
     exception: "SMS only available as a backup to Google Authenticator."
 
+  - name: Resolver
+    url: https://www.resolver.co.uk
+    img: resolver.png
+    twitter: resolvercouk
+    facebook: resolvercouk
+
```
Comment (Carlgo11) [inline _data/other.yml]:
```suggestion
    regions:
      - gb
      - in

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
Maybe `legal` would be a fitting category? 🤔

**Comment by ohheyrj** [comment]
Sure. I wasn't sure where to put it and since they help consumers with problems with many companies in different sectors id put it under other. Happy to move it to legal though

**Comment by ohheyrj** [comment]
Moved to legal as suggested :)

**Review by Carlgo11 — APPROVED** [review]
Great! Thanks 😄

**Review by phallobst — APPROVED** [review]

**Pull request merged**